build.gradle

```diff
@@ -1,34 +1,20 @@
+plugins {
+  id 'base'
+  id "io.spinnaker.project" version "$spinnakerGradleVersion" apply false
+  id "nebula.node" version "1.3.1"
+}
+
 import java.nio.file.Paths
 import groovy.json.JsonOutput
 import groovy.json.JsonSlurper
 import org.redline_rpm.header.Flags
+import com.netflix.spinnaker.gradle.publishing.bintray.BintrayPublishPlugin
 
-buildscript {
-  repositories {
-    jcenter()
-    maven { url "https://dl.bintray.com/spinnaker/gradle" }
-    maven { url "https://plugins.gradle.org/m2/" }
-  }
-  dependencies {
-    classpath 'com.netflix.spinnaker.gradle:spinnaker-gradle-project:4.2.0'
-    classpath 'com.moowork.gradle:gradle-node-plugin:1.1.1'
-  }
-}
 
 group = "com.netflix.spinnaker.deck"
-
-apply plugin: 'spinnaker.project'
-apply plugin: 'nebula.ospackage'
-apply plugin: 'com.moowork.node'
-apply plugin: 'java'
-apply plugin: 'com.github.hierynomus.license'
-
-jar {
-  dependsOn('webpack')
-  from(fileTree("build/webpack")) {
-    into "META-INF/resources"
-  }
-}
+apply plugin: "nebula.ospackage"
+project.tasks.register("publish")
```

Comment:
the publish lifecycle hook would normally exist from the publishing plugin, but we aren't applying that here. This is our entrypoint to building and publishing the deb.

BintrayPublishPlugin expects `publish` to exist so we have to register it first